homework/style.css

```diff
@@ -1,3 +1,145 @@
+* {
+  box-sizing: border-box;
```

Comment:
The select box run of the screen when viewed on a mobile phone.

![image](https://user-images.githubusercontent.com/2788771/66063617-05d6a280-e544-11e9-9b7a-e5f21d3f54fb.png)

Reply:
thank you @remarcmij , I will review css concepts, i need to improve some certain skilss in css